src/libsync/accessmanager.cpp

```diff
@@ -80,11 +80,15 @@ QNetworkReply *AccessManager::createRequest(QNetworkAccessManager::Operation op,
 
     // for some reason, passing an empty list causes the default chain to be removed
     // this behavior does not match the documentation
```

Comment:
This comment only makes sense right above the `if`.

Reply:
fixed in follow up